asv/benchmarks/setup/bench_model.py

```
@@ -143,6 +172,42 @@ def teardown(self, robot, num_worlds):
         del self._model
 
 
+class FastInitializeRender:
+    params = (["g1"], [256])
+    param_names = ["robot", "num_envs"]
```

Ruff (0.14.0)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)